controller.js

```diff
@@ -61,3 +62,17 @@ exports.getComments = (req, res, next) => {
     next("invalid id");
   }
 };
+
+exports.postComment = (req, res, next) => {
+  const { username, body } = req.body;
+  const article_id = Number(req.params.article_id);
+
+  writeContent(username, body, article_id)
+    .then((postedArticle) => {
+      res.status(201).send({ article: postedArticle[0] });
+    })
```

Comment:
We are posting a comment not an article. The name of this function "writeContent" may be the cause of confusion here and should be more descriptive. I would suggest "addComment" as an alternative name.